catchpy/settings/base.py

```diff
@@ -33,7 +35,7 @@
 if allowed_hosts_other:
     ALLOWED_HOSTS.extend(allowed_hosts_other.split())
 
-
+ALLOWED_HOSTS += get_ecs_task_ips()
```

Comment:
We should try to keep the base settings as generic as possible, so I would recommend moving the  `ALLOWED_HOSTS`  updates to `aws.py` instead. As long as `DJANGO_SETTINGS_MODULE=catchpy.settings.aws`, catchpy should automatically get all of the base settings plus the updates to the allowed hosts for an AWS ECS deployment. 

One additional benefit of making this change is that it eliminates the circular dependency that has been created since `base.py` and `aws.py` import each other:

```mermaid
graph TD;
    base-->aws;
    base-->test;
    aws-->base;
```

When what we want is this:

```mermaid
graph TD;
    base-->aws;
    base-->test;
```